tests/unit/workflows/test_workflows_utils.py

```diff
@@ -29,6 +29,7 @@
 import requests
 import requests_mock
 import time
+from flask import current_app
```

Comment:
this cannot work, in unit tests we don't have a flask app

Reply:
I thought so too, but actually they are..